idemp-rest-test/testing/src/tests/login.test.ts

```diff
@@ -0,0 +1,21 @@
+import { authenticationApi, IDEMPIERE_ENDPOINT } from '../api';
+import { AuthResponse } from '../types/org.bandahealth.idempiere.rest';
+
+test('can login', async () => {
+	await globalThis.__VALUE_OBJECT__.login();
+	expect(globalThis.__VALUE_OBJECT__.errorMessage).toBeFalsy();
+	expect(globalThis.__VALUE_OBJECT__.client).not.toBeFalsy();
+	expect(globalThis.__VALUE_OBJECT__.organization).not.toBeFalsy();
+	expect(globalThis.__VALUE_OBJECT__.role).not.toBeFalsy();
+	expect(globalThis.__VALUE_OBJECT__.warehouse).not.toBeFalsy();
+});
+
+test('200 returned if wrong username/password', async () => {
```

Comment:
wrong username/password returns 401 Unauthorized. thanks

Reply:
Lol well, it USED to return a 200. ;-) I'll update.

Reply:
Actually, it still does return 200. So, no update needed here and the test still passes.